src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionNoFinalizerTest.java

```diff
@@ -0,0 +1,64 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2020 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package org.checkstyle.suppressionxpathfilter;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.checks.coding.NoFinalizerCheck;
+
+public class XpathRegressionNoFinalizerTest extends AbstractXpathTestSupport {
+
+    @Override
+    protected String getCheckName() {
+        return NoFinalizerCheck.class.getSimpleName();
+    }
+
+    @Test
+    public void testOne() throws Exception {
```

Comment:
one tests is never enough. At least two is required.
please add caseof finalizer in inne class - https://github.com/checkstyle/checkstyle/blob/master/src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizeExtend.java#L45

Reply:
> case https://gaurabdg.github.io/checkstyle-tester-reports/NoFinalizer/diff/guava/index.html#A4
> it shows that violation is placed to not on line with method name , but on first annotation of method (where METHOD_DEF is started) - it is Ok for now. We place violation on same line as before.
> 
> items:

Ok, noted.

Reply:
> one tests is never enough. At least two is required.
> please add caseof finalizer in inne class - https://github.com/checkstyle/checkstyle/blob/master/src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizeExtend.java#L45 and place 2 annotations on method.

Done